src/components/common/connectionDialog/internetConnection.js

```diff
@@ -23,7 +23,7 @@ const InternetConnection = props => {
   const [showDisconnect, setShowDisconnect] = useState(false)
   const [firstLoadError, setFirstLoadError] = useState(true)
   const showDialogWindow = useCallback(
```

Comment:
do we need this function to be declared here with useCallback without dependencies?

Reply:
yes, with [] it will be defined only once, on mount. other wise the debounce will not work as it will be re-defined on every render